<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabAjouter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Ravie&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>631</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pren">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitter_ajout">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DateSAV">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_description">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>61</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCin">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_SAV</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>611</width>
         <height>401</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../aziz/png-clipart-til-death-do-us-apart-skull-illustration-t-shirt-clothing-skeleton-back-to-back-printing-tshirt-comics.png</pixmap>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>lineEdit_pren</zorder>
      <zorder>lineEdit_nom</zorder>
      <zorder>ajouter</zorder>
      <zorder>quitter_ajout</zorder>
      <zorder>lineEdit_DateSAV</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>lineEdit_description</zorder>
      <zorder>lineEditCin</zorder>
      <zorder>label_13</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>591</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>afficher</string>
      </property>
      <widget class="QPushButton" name="quitter_affich">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="afficher">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>501</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../aziz/wow mdp/firstquest.PNG</pixmap>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>groupBox_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>591</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_SAV</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCin_supp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quittersupp">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../aziz/wow mdp/firstquest.PNG</pixmap>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>591</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditCin_modif">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_modif">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitter_modif">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DateSAV_modif">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_description_modif">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pren_modif">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_SAV</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>861</width>
        <height>421</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../aziz/wow mdp/firstquest.PNG</pixmap>
      </property>
     </widget>
     <zorder>label_16</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Trier</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-10</y>
        <width>921</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../aziz/wow mdp/firstquest.PNG</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>601</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Trier</string>
      </property>
      <widget class="QTableView" name="tableView_tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>461</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_TriASC">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri ASC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_TriDESC">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>130</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri DESC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>recherche</string>
     </attribute>
     <widget class="QTableView" name="tableView_rechercher">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>451</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rechercher">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>stat</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>traitement</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>reboursement</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
